sdcm/cluster_gce.py

```diff
@@ -359,35 +365,16 @@ def _node_setup(self, node):
         node.remoter.run('sudo cat /etc/scylla.d/io.conf')
         node.wait_db_up()
         node.wait_jmx_up()
+        node.remoter.run('sudo yum install -y {}-gdb'.format(node.scylla_pkg()),
+                         verbose=verbose, ignore_status=True)
 
     def wait_for_init(self, node_list=None, verbose=False, timeout=None):
-        """
-        Configure scylla.yaml on all cluster nodes.
+        super(ScyllaGCECluster, self).wait_for_init(node_list=node_list, verbose=verbose, timeout=timeout)
 
-        We have to modify scylla.yaml on our own because we are not on AWS,
-        where there are auto config scripts in place.
-
-        :param node_list: List of nodes to watch for init.
-        :param verbose: Whether to print extra info while watching for init.
-        :param timeout: timeout in minutes to wait for init to be finished
-        :return:
-        """
-        if node_list is None:
-            node_list = self.nodes
-        res = self._wait_for_node_setup(node_list, verbose, timeout)
-        assert res is True, 'Node setup has failed!'
-
-        if not node_list[0].scylla_version:
-            result = node_list[0].remoter.run("rpm -q {}".format(node_list[0].scylla_pkg()))
-            match = re.findall("scylla-(.*).el7.centos", result.stdout)
-            node_list[0].scylla_version = match[0] if match else ''
-            for node in node_list:
-                node.scylla_version = result.stdout
-
-        for node in node_list:
-            dst_nodes = [n for n in node_list if n.dc_idx != node.dc_idx]
-            local_nodes = [n for n in node_list if n.dc_idx == node.dc_idx and n != node]
-            if self._param_enabled('enable_tc'):
+        if self._param_enabled('enable_tc'):
+            for node in self.nodes:
+                dst_nodes = [n for n in self.nodes if n.dc_idx != node.dc_idx]
+                local_nodes = [n for n in self.nodes if n.dc_idx == node.dc_idx and n != node]
                 self.set_tc(node, dst_nodes, local_nodes)
```

Comment:
@amoskong, a question about set_tc: this feature can be enabled for gce only?

Reply:
Yes for now.